portalnet/src/overlay_service.rs

```diff
@@ -351,15 +356,14 @@ where
         let (command_tx, command_rx) = mpsc::unbounded_channel();
         let internal_command_tx = command_tx.clone();
 
-        let overlay_protocol = protocol.clone();
-
         let peers_to_ping = if let Some(interval) = ping_queue_interval {
             HashSetDelay::new(interval)
         } else {
             HashSetDelay::default()
         };
 
         let (response_tx, response_rx) = mpsc::unbounded_channel();
+        let (event_stream, _) = broadcast::channel(10); // TODO: magic number
```

Comment:
Adding a constant would be useful here.

Reply:
Right. It's unclear whether 10 is the right choice for the capacity here though. Thoughts?

Reply:
I'm not sure but it seems like an okay starting point.

Reply:
Alright. I just pushed a change fixing the mentioned issues